### ISSUE TITLE
vg call report an warning and the output VCF file is empty

### ISSUE THREAD

**Issue by WWilen** [vgteam/vg#3509]
**1. What were you trying to do?**
Try to genotype SVs from the short-reads
'''
vg call wg.xg -k wg.pack -s QNM -v index/all.vcf.gz -t 10 -r Graph.snarls > TEST.vcf
'''
**2. What did you want to happen?**
get a output in VCF format

**3. What actually happened?**
All samples  reported:

 Warning: Site {"directed_acyclic_net_graph": true, "end": {"node_id": "1382750"}, "start": {"node_id": "1330320"}, "start_end_reachable": true, "type": 1} with 161 variants contains too many traversals (>50000) to enumerate so it will be skipped:  
......

for some samples ,the output VCF file is empty or incomplete(some chromosomes are missing) 

**4. If you got a line like `Stack trace path: /somewhere/on/your/computer/stacktrace.txt`, please copy-paste the contents of that file here:**

```
nothing
```

**5. What data and command can the vg dev team use to make the problem happen?**

**6. What does running `vg version` say?**

```
vg version v1.35.0 "Ghizzano"
Compiled with g++ (Ubuntu 10.3.0-1ubuntu1) 10.3.0 on Linux
Linked against libstd++ 20210408
Built by xian@sept

```

**Comment by glennhickey** [comment]
Hard to say from here.  Usually if the VCF is empty, it's because the pack file has very low coverage or there's a mismatch between the input files.  If you are able to share the input files I can take a look.

**Comment by WWilen** [comment]
Ah,I have some error information to add a bout empty VCF.

Pyrusbretschneideri_Z547-01-R03_good/vcf.sh: line 1: 283956 Killed                       vg call wg.xg -k Pyrusbretschneideri_Z547-01-R03_good/Pyrusbretschneideri_Z547-01-R03_good.pack -s Pyrusbretschneideri_Z547-01-R03_good -v index/All.vcf.gz -t 6 -r Graph.snarls > Pyrusbretschneideri_Z547-01-R03_good/Pyrusbretschneideri_Z547-01-R03_good.vcf

**Comment by glennhickey** [comment]
yeah, killing the command is a good way to get an empty vcf.  may have run out of memory.

**Comment by WWilen** [comment]
Ok，thank you for your help.One more question,Does it matter that c conains too many traversal to enumerate was skipped? I found that different sites were skipped in different samples.

**Comment by faithokamoto** [comment]
It will mean that you won't have results for there. You could try pre-simplifying the graph (e.g. by removing irrelevant paths) in advance if you want to avoid the issue.
